ui/src/shared/copy/notifications.ts

```diff
@@ -109,6 +109,11 @@ export const notifyCSVDownloadFailed = (): Notification => ({
   message: 'Unable to download .CSV file',
 })
 
```

Comment:
if `notifyViewerUnauthorizedToSetTempVars` is no longer being used, could you remove it from this file?

Reply:
done